setup.cfg

```diff
@@ -77,6 +80,6 @@ exclude: girder/external/*
 ignore: D100,D101,D102,D103,D104,D105,D106,D107,D200,D201,D202,D203,D204,D205,D400,D401,D402,E123,E226,E241,N802,N803,N806,N812,W503
 
 [tool:pytest]
-addopts = --verbose --strict --showlocals --cov=girder --cov-report=""
+addopts = --verbose --strict --showlocals --cov=girder --cov=plugins/jobs/server --cov-report=""
 cache_dir = build/test/pytest_cache
-testpaths = test
+testpaths = test plugins/jobs/test
```

Comment:
I was thinking about this the other day ... would it make sense to move the built-in plugin tests underneath the main "tests" directory, or do you think it is best to keep them under the plugin?

Reply:
That's a good point.  For now, it would probably make more sense to move them to `test/` to avoid this complication.  For Girder 3, I'm less convinced it is a good idea.

Reply:
We may also be able to use a globbing expression to avoid enumerating every plugin in the `--cov` flag.

Reply:
I just confirmed globbing expressions don't work for the `--cov` flag.